doc/manual/config-advanced.rst

```diff
@@ -355,3 +369,15 @@ contest ID to use, and the IDs of the team categories you want to include
 
 .. _Output Validators format: https://icpc.io/problem-package-format/spec/output_validators
 .. _default output validator: https://icpc.io/problem-package-format/spec/problem_package_format#default-output-validator-specification
+
+.. _clear-cache:
+
+Clearing the DOMjudge cache
+---------------------------
+
+Some operations require you to clear the DOMjudge cache. To do this, execute
+the `webapp/bin/console` (see the Config checker in the admin interfae for the
+full filesystem path of your installation) binary with the `cache:clear` subcommand::
+
+  webapp/bin/console cache:clear
+
```

Comment:
Make explicit that this is different from the scoreboard cache that you can also clear?